### PR TITLE
Call make build from setup and deploy

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: src/Resources/symfony/ansible/group_vars/deploy_demo.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - web/app_test.php
+
+#########
+# Tasks #
+#########
+
+manala_deploy_tasks:
+  - make: install@demo
+  - make: build@demo
+  - service_restart: supervisor
```
Comment (ogizanagi) [inline src/Resources/symfony/ansible/group_vars/deploy_demo.yml]: Is that required ? I don't have this in my project templates.

File: src/Resources/symfony/ansible/group_vars/deploy_demo.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
+---
+
+###########
+# Removed #
+###########
+
+manala_deploy_removed:
+  - web/app_dev.php
```
Comment (ogizanagi) [inline src/Resources/symfony/ansible/group_vars/deploy_demo.yml]: Should we remove this one by default on demo ? ping @nervo 

I'll personally be in favor of keeping it and only remove `app_test.php`

Comment (ogizanagi) [reply src/Resources/symfony/ansible/group_vars/deploy_demo.yml]: For reference: https://github.com/Elao/symfony-standard/blob/master/ansible/group_vars/deploy_demo.yml

Comment (chalasr) [reply src/Resources/symfony/ansible/group_vars/deploy_demo.yml]: I believe you're right, we do not remove it 👍

**Comment by chalasr** [comment]
Done

**Review by ogizanagi — APPROVED** [review]

**Pull request merged**